courses/intro-to-ada/chapters/standard_library_files_streams.rst

```diff
@@ -0,0 +1,453 @@
+Standard library: Files and streams
+===================================
+
+:code-config:`reset_accumulator=True;accumulate_code=False`
+
+.. include:: ../../global.txt
+
+This chapter presents the different options available in Ada for file
```

Comment:
You changed "This section" to "This chapter".
None of the other pieces use this self-reference (so they didn't have to change anything) so maybe if you're tweaking it anyways, you could make this opening paragraph more like the other pieces? Just a thought